packages/xrpl/snippets/src/claimPayChannel.ts

```diff
@@ -4,7 +4,8 @@ import {
   PaymentChannelCreate,
   PaymentChannelClaim,
   hashes,
-} from '../../dist/npm'
+  // Change to 'xrpl' if you are using this code outside of the xrpl.js repo
+} from 'xrpl-local'
```

Comment:
This feels more confusing. Could we instead just use `xrpl` and build/test the snippets separately?

Reply:
We could, but that would need to be a separate ticket

Reply:
This also came up in my linter update PR here: https://github.com/XRPLF/xrpl.js/pull/2168

I changed the `tsconfig` to let us point directly to `src` files instead of using `xrpl-local`. Not sure which method is better, but probably using `xrpl` directly is best all things considered.

Reply:
`src` is better than `xrpl-local` - it's less confusing

Reply:
I just added an alias to package.json for devDependencies

"xrpl": "file:./src",
"xrpl-local": "file:./src"

thoughts?

Reply:
That seems reasonable, but should go into a separate ticket, and just update xrpl-local -> xrpl if that works :)